2024-03-13 20:32:31,087 INFO    StreamThr :25808 [internal.py:wandb_internal():86] W&B internal server running at pid: 25808, started at: 2024-03-13 20:32:31.087047
2024-03-13 20:32:31,088 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status
2024-03-13 20:32:31,094 INFO    WriterThread:25808 [datastore.py:open_for_write():85] open: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\run-tk3r2dy7.wandb
2024-03-13 20:32:31,095 DEBUG   SenderThread:25808 [sender.py:send():380] send: header
2024-03-13 20:32:31,095 DEBUG   SenderThread:25808 [sender.py:send():380] send: run
2024-03-13 20:32:32,393 INFO    SenderThread:25808 [dir_watcher.py:__init__():211] watching files in: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files
2024-03-13 20:32:32,394 INFO    SenderThread:25808 [sender.py:_start_run_threads():1122] run started: tk3r2dy7 with start time 1710333151.087046
2024-03-13 20:32:32,412 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: check_version
2024-03-13 20:32:32,413 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: check_version
2024-03-13 20:32:35,582 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: run_start
2024-03-13 20:32:35,623 DEBUG   HandlerThread:25808 [system_info.py:__init__():32] System info init
2024-03-13 20:32:35,624 DEBUG   HandlerThread:25808 [system_info.py:__init__():47] System info init done
2024-03-13 20:32:35,624 INFO    HandlerThread:25808 [system_monitor.py:start():181] Starting system monitor
2024-03-13 20:32:35,624 INFO    SystemMonitor:25808 [system_monitor.py:_start():145] Starting system asset monitoring threads
2024-03-13 20:32:35,624 INFO    HandlerThread:25808 [system_monitor.py:probe():201] Collecting system info
2024-03-13 20:32:35,629 INFO    SystemMonitor:25808 [interfaces.py:start():190] Started cpu monitoring
2024-03-13 20:32:35,630 INFO    SystemMonitor:25808 [interfaces.py:start():190] Started disk monitoring
2024-03-13 20:32:35,630 INFO    SystemMonitor:25808 [interfaces.py:start():190] Started gpu monitoring
2024-03-13 20:32:35,631 INFO    SystemMonitor:25808 [interfaces.py:start():190] Started memory monitoring
2024-03-13 20:32:35,645 INFO    SystemMonitor:25808 [interfaces.py:start():190] Started network monitoring
2024-03-13 20:32:35,668 DEBUG   HandlerThread:25808 [system_info.py:probe():196] Probing system
2024-03-13 20:32:35,668 DEBUG   HandlerThread:25808 [system_info.py:probe():244] Probing system done
2024-03-13 20:32:35,669 DEBUG   HandlerThread:25808 [system_monitor.py:probe():210] {'os': 'Windows-10-10.0.22000-SP0', 'python': '3.10.9', 'heartbeatAt': '2024-03-13T12:32:35.668881', 'startedAt': '2024-03-13T12:32:31.072756', 'docker': None, 'cuda': None, 'args': ('--task=P', '--weather=foggy'), 'state': 'running', 'program': 'E:\\自动驾驶\\车位检测\\parking-space-Identification-in-different-weather-main\\train.py', 'codePathLocal': 'train.py', 'codePath': 'train.py', 'host': 'LAPTOP-QIVT4NMT', 'username': 'lenovo', 'executable': 'E:\\anaconda\\python.exe', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 3201.0, 'min': 0.0, 'max': 3201.0}, 'cpu_freq_per_core': [{'current': 3201.0, 'min': 0.0, 'max': 3201.0}], 'disk': {'/': {'total': 465.74511337280273, 'used': 178.2540626525879}}, 'gpu': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'memory_total': 6442450944}], 'memory': {'total': 15.855525970458984}}
2024-03-13 20:32:35,669 INFO    HandlerThread:25808 [system_monitor.py:probe():211] Finished collecting system info
2024-03-13 20:32:35,669 INFO    HandlerThread:25808 [system_monitor.py:probe():214] Publishing system info
2024-03-13 20:32:35,669 DEBUG   HandlerThread:25808 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-03-13 20:32:35,670 DEBUG   HandlerThread:25808 [system_info.py:_save_pip():68] Saving pip packages done
2024-03-13 20:32:35,670 DEBUG   HandlerThread:25808 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-03-13 20:32:35,679 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:36,426 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\conda-environment.yaml
2024-03-13 20:32:36,426 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\requirements.txt
2024-03-13 20:32:37,711 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:39,746 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:41,795 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:42,460 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\conda-environment.yaml
2024-03-13 20:32:42,575 DEBUG   HandlerThread:25808 [system_info.py:_save_conda():87] Saving conda packages done
2024-03-13 20:32:42,576 INFO    HandlerThread:25808 [system_monitor.py:probe():216] Finished publishing system info
2024-03-13 20:32:42,596 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:32:42,596 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: keepalive
2024-03-13 20:32:42,596 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:32:42,597 DEBUG   SenderThread:25808 [sender.py:send():380] send: files
2024-03-13 20:32:42,597 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-metadata.json with policy now
2024-03-13 20:32:42,608 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:32:42,608 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:32:42,610 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:32:43,311 DEBUG   SenderThread:25808 [sender.py:send():380] send: telemetry
2024-03-13 20:32:43,464 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\conda-environment.yaml
2024-03-13 20:32:43,465 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-metadata.json
2024-03-13 20:32:43,465 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:32:43,833 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:45,493 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:32:45,875 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:47,350 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:32:47,912 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:49,557 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:32:49,558 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:32:49,558 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:32:49,559 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:32:49,939 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:50,545 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:32:51,297 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:32:51,298 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:32:51,298 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:32:51,299 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:32:51,300 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:32:51,300 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:32:51,300 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:32:51,301 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:32:51,551 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:32:51,552 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:32:51,965 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:53,097 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825C12680>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:32:53,097 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 20:32:53,097 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:32:53,317 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:32:53,556 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:32:53,999 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:55,190 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:32:55,191 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:32:55,191 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:32:55,192 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:32:55,565 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:32:56,037 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:56,949 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:32:56,950 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:32:56,951 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:32:56,951 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:32:56,952 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:32:56,953 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:32:56,953 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:32:56,954 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:32:57,570 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:32:57,571 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:32:57,601 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:32:57,617 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:32:57,618 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:32:58,078 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:32:59,003 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:00,117 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:00,878 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:00,879 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:00,879 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:00,880 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:01,590 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:01,591 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:02,150 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:02,633 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:02,634 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:02,635 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:02,635 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:02,636 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:02,637 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:02,637 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:02,637 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:03,601 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:03,602 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:04,190 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:04,657 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:05,156 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E171C0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:33:05,156 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 20:33:05,156 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:33:06,278 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:06,529 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:06,530 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:06,530 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:06,531 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:06,618 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\config.yaml
2024-03-13 20:33:06,619 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:07,621 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:08,290 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:08,290 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:08,291 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:08,292 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:08,293 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:08,293 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:08,293 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:08,294 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:08,309 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:08,626 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:09,628 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:10,331 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:12,186 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:12,187 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:12,187 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:12,187 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:12,188 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:12,350 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:12,617 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:33:12,617 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:33:12,617 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:33:12,665 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:13,667 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:13,939 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:13,940 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:13,941 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:13,941 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:13,942 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:13,942 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:13,942 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:13,943 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:14,385 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:14,681 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:15,684 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:16,422 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:17,660 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E331F0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:33:17,661 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 20:33:17,661 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:33:17,661 INFO    wandb-upload_0:25808 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "E:\anaconda\lib\site-packages\urllib3\util\connection.py", line 95, in create_connection
    raise err
  File "E:\anaconda\lib\site-packages\urllib3\util\connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 714, in urlopen
    httplib_response = self._make_request(
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 403, in _make_request
    self._validate_conn(conn)
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 1053, in _validate_conn
    conn.connect()
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x0000015825E331F0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 487, in send
    resp = conn.urlopen(
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 798, in urlopen
    retries = retries.increment(
  File "E:\anaconda\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E331F0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E331F0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\wandb\sdk\lib\retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2580, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-03-13 20:33:17,838 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:17,838 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:17,839 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:17,839 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:17,840 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:18,453 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:18,693 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:19,562 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:19,563 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:19,564 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:19,564 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:19,565 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:19,565 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:19,566 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:19,566 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:19,698 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:19,699 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:20,495 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:21,709 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:22,535 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:23,440 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:23,440 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:23,441 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:23,441 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:23,442 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:23,721 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:24,582 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:25,163 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:25,164 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:25,165 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:25,165 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:25,166 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:25,167 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:25,167 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:25,168 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:25,739 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:25,740 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:26,613 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:27,622 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:33:27,638 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:33:27,639 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:33:27,751 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:28,663 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:29,023 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:29,083 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:29,083 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:29,084 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:29,084 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:29,758 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:29,759 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:30,697 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:30,850 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:30,850 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:30,851 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:30,852 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:30,852 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:30,853 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:30,853 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:30,854 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:31,769 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:31,770 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:32,608 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E178B0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:33:32,609 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 20:33:32,609 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:33:32,728 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:34,769 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:34,769 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:34,770 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:34,770 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:34,770 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:34,771 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:34,783 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:35,673 DEBUG   SystemMonitor:25808 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2024-03-13 20:33:35,674 DEBUG   SenderThread:25808 [sender.py:send():380] send: stats
2024-03-13 20:33:35,786 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:36,468 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:36,469 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:36,470 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:36,470 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:36,471 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:36,471 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:36,471 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:36,472 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:36,799 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:36,857 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:37,795 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:38,881 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:40,365 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:40,366 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:40,366 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:40,366 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:40,368 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:40,869 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:40,902 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:41,876 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:42,073 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:42,073 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:42,074 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:42,075 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:42,075 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:42,076 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:42,076 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:42,076 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:42,635 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:33:42,636 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:33:42,636 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:33:42,901 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:42,921 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:43,903 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:44,958 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:45,959 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:45,960 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:45,960 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:45,961 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:45,961 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:46,966 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:46,986 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:47,720 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:47,721 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:47,721 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:47,722 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:47,723 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:47,723 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:47,723 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:47,724 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:47,969 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:47,969 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:49,017 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:50,006 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:51,056 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:51,618 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:51,618 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:51,619 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:51,619 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:51,620 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:51,993 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E312D0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:33:51,994 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 20:33:51,994 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:33:52,047 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:53,095 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:53,356 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:53,356 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:53,357 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:53,357 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:53,358 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:53,359 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:53,359 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:53,359 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:54,090 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:54,091 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:55,136 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:56,132 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:33:57,172 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:33:57,232 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:33:57,233 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:57,233 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:57,234 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:57,234 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:57,649 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:33:57,649 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:33:57,649 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:33:58,159 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:58,946 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:58,947 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:33:58,947 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:58,948 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:58,949 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:58,949 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:33:58,949 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:33:58,950 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:33:59,178 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:33:59,199 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:00,187 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:01,233 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:02,831 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:02,832 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:02,832 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:02,833 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:02,834 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:03,252 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:03,270 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:04,267 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:04,575 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:04,575 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:04,576 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:04,577 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:04,577 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:04,578 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:04,578 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:04,579 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:05,284 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:05,300 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:05,683 DEBUG   SenderThread:25808 [sender.py:send():380] send: stats
2024-03-13 20:34:06,288 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:07,389 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:08,452 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:08,453 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:08,453 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:08,454 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:08,454 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:09,358 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:09,410 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:10,203 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:10,204 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:10,204 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:10,205 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:10,206 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:10,206 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:10,206 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:10,207 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:10,365 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:10,366 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:11,433 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:12,403 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:12,657 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:34:12,658 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:34:12,658 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:34:13,455 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:14,025 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:14,088 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:14,089 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:14,089 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:14,090 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:14,440 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:15,487 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:15,801 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:15,802 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:15,803 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:15,803 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:15,804 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:15,804 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:15,804 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:15,805 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:16,479 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:16,480 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:17,525 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:18,512 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:19,564 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:19,690 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:19,690 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:19,691 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:19,691 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:19,692 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:20,556 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:20,652 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-metadata.json?Expires=1710419565&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DhUZ77j3qG7v7bDdwMvRS50s5FctckwQlak0A7j7MuhrFc%2FPDkwME87Pm5utPILiXsMgx6CxIB2UFGoF%2FXs85iyIOroXdfXwIf0bdqLqHjIXmb6DVqP8FmpfntyXsxX11oa%2FtMBycNGS%2BOma0J8%2BrMNpxSmGMsJf7NzfLZsgfFcWGzLmRbHqgnkeiZQCKmryG45wAq9sae0V5jRMgGiqx61SREylJdA%2BM6UhFP3ZRKm99j89zrsfXANX73cd%2BQ49HgkWZpsVj531URimsGguIdD4JFg9Cntx2z%2FY810tbq0Ap6SorExd5CtIwp%2Bj5C%2F1sEXJvGuuhGz5p3rtVxny5A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E33490>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:34:20,653 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 20:34:20,653 ERROR   wandb-upload_0:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:34:21,455 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:21,455 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:21,456 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:21,457 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:21,457 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:21,458 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:21,458 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:21,459 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:21,575 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:21,605 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:22,583 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:23,643 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:25,344 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:25,345 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:25,345 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:25,345 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:25,346 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:25,656 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:25,672 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:27,050 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:27,051 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:27,051 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:27,052 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:27,053 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:27,053 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:27,053 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:27,054 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:27,685 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:34:27,702 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:34:27,703 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:34:27,705 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:27,714 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:28,707 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:29,749 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:31,066 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:31,067 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:31,067 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:31,067 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:31,068 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:31,784 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:31,784 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:31,804 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:32,812 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:32,812 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:32,813 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:32,813 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:32,814 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:32,815 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:32,815 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:32,815 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:33,820 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:33,821 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:33,839 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:35,689 DEBUG   SenderThread:25808 [sender.py:send():380] send: stats
2024-03-13 20:34:35,882 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:36,692 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:36,815 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:36,815 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:36,815 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:36,816 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:36,869 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:37,915 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:37,968 ERROR   gpu       :25808 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:34:38,586 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:38,586 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:34:38,587 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:38,587 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:38,588 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:38,589 DEBUG   SenderThread:25808 [sender.py:send():380] send: history
2024-03-13 20:34:38,589 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:34:38,589 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:38,732 DEBUG   SenderThread:25808 [sender.py:send():380] send: exit
2024-03-13 20:34:38,732 INFO    SenderThread:25808 [sender.py:send_exit():585] handling exit code: 0
2024-03-13 20:34:38,733 INFO    SenderThread:25808 [sender.py:send_exit():587] handling runtime: 123
2024-03-13 20:34:38,733 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:38,734 INFO    SenderThread:25808 [sender.py:send_exit():593] send defer
2024-03-13 20:34:38,734 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:38,734 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 0
2024-03-13 20:34:38,734 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:38,734 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 0
2024-03-13 20:34:38,734 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 1
2024-03-13 20:34:38,734 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:38,735 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 1
2024-03-13 20:34:38,735 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:38,735 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 1
2024-03-13 20:34:38,735 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 2
2024-03-13 20:34:38,735 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:38,735 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 2
2024-03-13 20:34:38,735 INFO    HandlerThread:25808 [system_monitor.py:finish():190] Stopping system monitor
2024-03-13 20:34:38,735 DEBUG   SystemMonitor:25808 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2024-03-13 20:34:38,742 DEBUG   SystemMonitor:25808 [system_monitor.py:_start():170] Publishing last batch of metrics
2024-03-13 20:34:38,743 INFO    HandlerThread:25808 [interfaces.py:finish():202] Joined cpu monitor
2024-03-13 20:34:38,743 INFO    HandlerThread:25808 [interfaces.py:finish():202] Joined disk monitor
2024-03-13 20:34:38,784 INFO    HandlerThread:25808 [interfaces.py:finish():202] Joined gpu monitor
2024-03-13 20:34:38,784 INFO    HandlerThread:25808 [interfaces.py:finish():202] Joined memory monitor
2024-03-13 20:34:38,784 INFO    HandlerThread:25808 [interfaces.py:finish():202] Joined network monitor
2024-03-13 20:34:38,785 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:38,785 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 2
2024-03-13 20:34:38,785 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 3
2024-03-13 20:34:38,785 DEBUG   SenderThread:25808 [sender.py:send():380] send: stats
2024-03-13 20:34:38,785 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:38,786 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 3
2024-03-13 20:34:38,786 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:38,786 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 3
2024-03-13 20:34:38,786 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 4
2024-03-13 20:34:38,786 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:38,786 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 4
2024-03-13 20:34:38,787 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:38,787 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 4
2024-03-13 20:34:38,787 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 5
2024-03-13 20:34:38,787 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:38,787 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 5
2024-03-13 20:34:38,787 DEBUG   SenderThread:25808 [sender.py:send():380] send: summary
2024-03-13 20:34:38,788 INFO    SenderThread:25808 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:34:38,789 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:38,789 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 5
2024-03-13 20:34:38,789 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 6
2024-03-13 20:34:38,789 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:38,789 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 6
2024-03-13 20:34:38,789 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:38,789 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 6
2024-03-13 20:34:38,789 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 7
2024-03-13 20:34:38,789 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:38,789 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:38,790 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 7
2024-03-13 20:34:38,790 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:38,790 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 7
2024-03-13 20:34:38,917 INFO    Thread-12 :25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json
2024-03-13 20:34:39,737 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:39,847 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 8
2024-03-13 20:34:39,847 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:39,847 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:39,847 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 8
2024-03-13 20:34:39,848 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:39,848 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 8
2024-03-13 20:34:39,848 INFO    SenderThread:25808 [job_builder.py:build():281] Attempting to build job artifact
2024-03-13 20:34:39,849 INFO    SenderThread:25808 [job_builder.py:_get_source_type():404] no source found
2024-03-13 20:34:39,849 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 9
2024-03-13 20:34:39,849 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:39,849 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 9
2024-03-13 20:34:39,849 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:39,849 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 9
2024-03-13 20:34:39,849 INFO    SenderThread:25808 [dir_watcher.py:finish():358] shutting down directory watcher
2024-03-13 20:34:39,929 INFO    SenderThread:25808 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log
2024-03-13 20:34:39,929 INFO    SenderThread:25808 [dir_watcher.py:finish():388] scan: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files
2024-03-13 20:34:39,929 INFO    SenderThread:25808 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\conda-environment.yaml conda-environment.yaml
2024-03-13 20:34:39,930 INFO    SenderThread:25808 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\config.yaml config.yaml
2024-03-13 20:34:39,932 INFO    SenderThread:25808 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\output.log output.log
2024-03-13 20:34:39,934 INFO    SenderThread:25808 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\requirements.txt requirements.txt
2024-03-13 20:34:39,936 INFO    SenderThread:25808 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-metadata.json wandb-metadata.json
2024-03-13 20:34:39,937 INFO    SenderThread:25808 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_203231-tk3r2dy7\files\wandb-summary.json wandb-summary.json
2024-03-13 20:34:39,939 INFO    SenderThread:25808 [sender.py:transition_state():613] send defer: 10
2024-03-13 20:34:39,939 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:34:39,939 INFO    HandlerThread:25808 [handler.py:handle_request_defer():170] handle defer: 10
2024-03-13 20:34:39,941 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: defer
2024-03-13 20:34:39,941 INFO    SenderThread:25808 [sender.py:send_request_defer():609] handle sender defer: 10
2024-03-13 20:34:39,941 INFO    SenderThread:25808 [file_pusher.py:finish():175] shutting down file pusher
2024-03-13 20:34:40,746 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:40,747 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:41,755 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:41,756 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:42,760 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:42,760 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:43,767 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:43,768 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:44,777 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:44,777 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:45,791 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:45,792 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:46,802 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:46,802 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:46,803 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:47,814 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:47,815 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:48,822 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:48,822 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:49,826 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:49,827 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:49,839 ERROR   wandb-upload_2:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/config.yaml?Expires=1710419682&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=evyt04Sh935%2Fo5srHeFGfe8CLe9iyO80Dy818JtBz94mwkOu1X3xEdkkUHE34vYM5myCXns%2B21GQgaOhjFRicK8hNAwIKlyuujNfCftYxs9rCThnG535evgq7CT5uJAomoW6RsGBZV4%2BDv8uP96WSVH%2FeafRr6kUhPuSOqc30HAkZ4CtP5T3ruZMaFNx99eaofcbsbGYy5ccVKIow%2FtBD00gV9vRaEsINmLECtAQa9mkxpT1Ctx3tGrOXF%2Bun5Hyb4uvs8JXULxlCnSbSkcI1YPEKrK%2BPhKCZrLOT6rIcpwwobbwg9e2%2Fhq225fF617gV%2FDCLb1PCqk4UhJll7FHsA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/config.yaml?Expires=1710419682&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=evyt04Sh935%2Fo5srHeFGfe8CLe9iyO80Dy818JtBz94mwkOu1X3xEdkkUHE34vYM5myCXns%2B21GQgaOhjFRicK8hNAwIKlyuujNfCftYxs9rCThnG535evgq7CT5uJAomoW6RsGBZV4%2BDv8uP96WSVH%2FeafRr6kUhPuSOqc30HAkZ4CtP5T3ruZMaFNx99eaofcbsbGYy5ccVKIow%2FtBD00gV9vRaEsINmLECtAQa9mkxpT1Ctx3tGrOXF%2Bun5Hyb4uvs8JXULxlCnSbSkcI1YPEKrK%2BPhKCZrLOT6rIcpwwobbwg9e2%2Fhq225fF617gV%2FDCLb1PCqk4UhJll7FHsA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E32200>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:34:49,839 ERROR   wandb-upload_2:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '456'}
2024-03-13 20:34:49,839 ERROR   wandb-upload_2:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:34:50,236 ERROR   wandb-upload_5:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-summary.json?Expires=1710419683&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DogpXkn5VGpJ7uqsqNITudyUapFSRdAzPrq2rtL7ynpJiEvi7a85cTZjxTse9m7qsCHRpQS5vzo6R2%2FeraLuogpgDYOBWZGIC6VtLKiOmW8aiKzttncc5g3Nx2WkpLaghe9Kx3CRPceuPTGobwAGLcbVaYtA9z1biVi7Pab2wPM00t3PC4yw%2FHu1i9gOnOXrahh4slfe9dr39Y0ILg44r1tsrAPTBRJb%2FYvlfmEjarDawhTqpXGR2SbkHneWYEj2HKV6Qzi61BMk4eHjiM5nOkC3UMr%2FP1elMluO%2F%2Fh6%2Bs1N%2BnC50bOQOt7%2FHEy7cYKYMDX6f39WEv4FzbcGrSfg7Q%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/wandb-summary.json?Expires=1710419683&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DogpXkn5VGpJ7uqsqNITudyUapFSRdAzPrq2rtL7ynpJiEvi7a85cTZjxTse9m7qsCHRpQS5vzo6R2%2FeraLuogpgDYOBWZGIC6VtLKiOmW8aiKzttncc5g3Nx2WkpLaghe9Kx3CRPceuPTGobwAGLcbVaYtA9z1biVi7Pab2wPM00t3PC4yw%2FHu1i9gOnOXrahh4slfe9dr39Y0ILg44r1tsrAPTBRJb%2FYvlfmEjarDawhTqpXGR2SbkHneWYEj2HKV6Qzi61BMk4eHjiM5nOkC3UMr%2FP1elMluO%2F%2Fh6%2Bs1N%2BnC50bOQOt7%2FHEy7cYKYMDX6f39WEv4FzbcGrSfg7Q%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E33790>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:34:50,236 ERROR   wandb-upload_1:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/conda-environment.yaml?Expires=1710419682&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=ViRiQebP7xn7iWZWAftkOTCUqwpiaVuYV1iCziA6RNeX4v0dpRdCn3gfzslCN8u%2F%2Bc4rMVWTg7X2Ipslu0Q42GSu4dSnARYitJdsoRZgv3qWf9GtuFDDKrbE8xS2GGal8MAgnsuLdklkW1%2FjlGL3oHjZUSqzIt7xRXIQLnw4IIbB2frj1B3T6qT7tHcrba065GmHgGYYXyjerPn8Y1vZlWi3tlSq87H54AEv1mY0CN1Dz42iat4SjExWFaTlAxWR4zWsI9x8Ys1TkMWG50VOypvSk0TZFm6D0AwZiEIkodcqGIZY24y0lbkUhXugfkAXWHw6T75ZHg%2Fi4lWCCqr9vg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/conda-environment.yaml?Expires=1710419682&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=ViRiQebP7xn7iWZWAftkOTCUqwpiaVuYV1iCziA6RNeX4v0dpRdCn3gfzslCN8u%2F%2Bc4rMVWTg7X2Ipslu0Q42GSu4dSnARYitJdsoRZgv3qWf9GtuFDDKrbE8xS2GGal8MAgnsuLdklkW1%2FjlGL3oHjZUSqzIt7xRXIQLnw4IIbB2frj1B3T6qT7tHcrba065GmHgGYYXyjerPn8Y1vZlWi3tlSq87H54AEv1mY0CN1Dz42iat4SjExWFaTlAxWR4zWsI9x8Ys1TkMWG50VOypvSk0TZFm6D0AwZiEIkodcqGIZY24y0lbkUhXugfkAXWHw6T75ZHg%2Fi4lWCCqr9vg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E33670>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:34:50,237 ERROR   wandb-upload_5:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '209'}
2024-03-13 20:34:50,237 ERROR   wandb-upload_1:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9492'}
2024-03-13 20:34:50,237 ERROR   wandb-upload_5:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:34:50,237 ERROR   wandb-upload_1:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:34:50,519 ERROR   wandb-upload_3:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/output.log?Expires=1710419683&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=JKyi5JtG%2Fv6sq2Gg9O%2BJ1NHGzEH4HGvHuD4QAOMk9CBxoRraPdbdLD14y1l0H9kX7Kvz5Zub4Myoo95IYvSlu1JDdp92bDyzFI4ze6mQRaoXvZJ1npPm23dIvLMFLogJ%2BgRADwitAX3TmsSvy5DlLzQQh3xbXK5tKnorUEVJ9S5CypD0BujdJ5BQdsUvs9Xfdp%2BLxIHwNg%2FuuFDYnAWYpupSeIRk3B2BkG0ITQKVpuWVOUWCKy9ApYSVRRFq1Cfx9mu49efE9WBzbz%2BlpZ0vGQAw6NKgwELDJfuy4XkJMYmhbC2zsbpvx%2BEvRVwOOi3f6RWaBbC28dDiiaTDZnXsSg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/output.log?Expires=1710419683&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=JKyi5JtG%2Fv6sq2Gg9O%2BJ1NHGzEH4HGvHuD4QAOMk9CBxoRraPdbdLD14y1l0H9kX7Kvz5Zub4Myoo95IYvSlu1JDdp92bDyzFI4ze6mQRaoXvZJ1npPm23dIvLMFLogJ%2BgRADwitAX3TmsSvy5DlLzQQh3xbXK5tKnorUEVJ9S5CypD0BujdJ5BQdsUvs9Xfdp%2BLxIHwNg%2FuuFDYnAWYpupSeIRk3B2BkG0ITQKVpuWVOUWCKy9ApYSVRRFq1Cfx9mu49efE9WBzbz%2BlpZ0vGQAw6NKgwELDJfuy4XkJMYmhbC2zsbpvx%2BEvRVwOOi3f6RWaBbC28dDiiaTDZnXsSg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E173D0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:34:50,519 ERROR   wandb-upload_3:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1436'}
2024-03-13 20:34:50,519 ERROR   wandb-upload_3:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:34:50,831 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:50,831 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:51,342 ERROR   wandb-upload_4:25808 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/requirements.txt?Expires=1710419683&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=hwEEpdekrRNCNK%2Br3n2ssVjLL%2FTPc7ISFWtz6sRWG1qE8Ko1EgRu6C8TCMuhJ7OP4LEbl7kBZxwdpV%2FTJ9RHyptN4hh9JpdKKdCpwuqsjhnzkRx3IofpeabW291MG8axN%2FKWZHGqUEuzbN4CEn3dnn5yJ2Mm%2FPzmujTa2p9sEHPxFHkxdJPbrF4l%2BJ7ofA9r26JkRM8tU4LLEuz91eR6pnrTO1kqW9osdAzWXce0Y0Dc5%2B2VXM7J5hZppeplDTxzzUOIuvulKXLUI6wGSY0gBGGBb6bIT8DD6Bf7%2BWU5C4kakVG9A7Nry%2BGO4c4Sp8KhEbdmzs1zWQBlW4JEkhT6bg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/tk3r2dy7/requirements.txt?Expires=1710419683&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=hwEEpdekrRNCNK%2Br3n2ssVjLL%2FTPc7ISFWtz6sRWG1qE8Ko1EgRu6C8TCMuhJ7OP4LEbl7kBZxwdpV%2FTJ9RHyptN4hh9JpdKKdCpwuqsjhnzkRx3IofpeabW291MG8axN%2FKWZHGqUEuzbN4CEn3dnn5yJ2Mm%2FPzmujTa2p9sEHPxFHkxdJPbrF4l%2BJ7ofA9r26JkRM8tU4LLEuz91eR6pnrTO1kqW9osdAzWXce0Y0Dc5%2B2VXM7J5hZppeplDTxzzUOIuvulKXLUI6wGSY0gBGGBb6bIT8DD6Bf7%2BWU5C4kakVG9A7Nry%2BGO4c4Sp8KhEbdmzs1zWQBlW4JEkhT6bg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000015825E33A30>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:34:51,343 ERROR   wandb-upload_4:25808 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3463'}
2024-03-13 20:34:51,343 ERROR   wandb-upload_4:25808 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:34:51,840 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:51,841 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:51,841 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:52,855 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:52,855 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:53,866 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:53,866 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:54,879 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:54,879 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:55,893 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:55,894 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:34:56,908 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:34:56,909 DEBUG   HandlerThread:25808 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:34:56,909 DEBUG   SenderThread:25808 [sender.py:send_request():407] send_request: poll_exit
